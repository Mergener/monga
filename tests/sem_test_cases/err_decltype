function foo(): int {
	var x : int;
	var f : someType;

	x = f.value;
	return x;
}