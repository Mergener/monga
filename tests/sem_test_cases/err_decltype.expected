[Error] Module tests/sem_test_cases/err_decltype, line 3, column 19: Unknown type 'someType' for variable 'f'.
Semantic errors were caught during analysis.
